File: solutions/julia/secret-handshake/2/secret-handshake.jl
function secret_handshake(code)
    actions = ["wink", "double blink", "close your eyes", "jump"]
    bits = reverse(bitstring(UInt16(code)))
    output = filter(x->x!=false,[bits[i]=='1' && actions[i] for i in 1:4])
    bits[5] == '1' && return reverse(output)
    output                                
end

File: solutions/python/secret-handshake/2/secret_handshake.py
def commands(binary_str):
    actions = ["wink", "double blink", "close your eyes", "jump"]
    actions.reverse()
    output = []
    for idx in range(len(binary_str)-1,0,-1):
        if binary_str[idx] == "1":
            output.append(actions[idx-1])
    if binary_str[0]=="1":
            output.reverse()
    return output
            
